tests/e2e/cli/instance_segmentation/test_instance_segmentation.py

```diff
@@ -152,7 +152,7 @@ def test_otx_eval(self, template, tmp_dir_path):
     @pytest.mark.parametrize("half_precision", [True, False])
     def test_otx_eval_openvino(self, template, tmp_dir_path, half_precision):
         tmp_dir_path = tmp_dir_path / "ins_seg"
-        otx_eval_openvino_testing(template, tmp_dir_path, otx_dir, args, threshold=0.2, half_precision=half_precision)
+        otx_eval_openvino_testing(template, tmp_dir_path, otx_dir, args, threshold=0.1, half_precision=half_precision)
```

Comment:
All other algo's ov eval tasks have 0.2 threshold. I think we don't need to change this. And can we remove skip mark from this update?

Reply:
I reverted the threshold. Unfortunately, skip marks are still valid